lib/docker-compose.yml

```diff
@@ -105,6 +119,25 @@ services:
         ipv4_address: $PMA_IP
     ports:
       - "$PUBLIC_PMA_PORT:80"
+  sshfs:
```

Comment:
le fait d'embarquer sshfs dans Drucker n'a pas de sens d'un point de vue Opensource. C'est une problematique interne. 
il faut l'enlever d'ici, et faire un script qui rajoute on-the-fly un sshfs au project docker lancé sur la sdev (ou équivalent).